test/quantize_cpu.cpp

```diff
@@ -46,10 +46,10 @@ void DequantizeTest(int input, float min, float max, float expected_output)
     auto f = make_shared<Function>(r, op::ParameterVector{A});
     auto backend = runtime::Backend::create("CPU");
     // Create some tensors for input/output
-    auto a = backend->create_tensor(element::from<T>(), Shape{});
+    auto a = backend->create_tensor(element::from<T>(), Shape{1});
```

Comment:
why shape{1}?

Reply:
`shape_a` has shape {1}. Could also make `shape_a` have shape {} and leave the others go I think.

Reply:
yes, we can make shape_a have shape{} and use shape_a everywhere.  my bad that the test still has shape_a as Shape{1}

Reply:
There's an upcoming PR for making sure layouts assigned to a tensor agree with the shape of the tensor, and it caught this test.

Reply:
Shape{1} is needed to get things into MKLDNN. Currently MKLDNN doesn't like zero-dim tensors.

Reply:
ah ok ya, I hit the same issue in pooling as well, they do support it in the newer release though